e2e/libs/k8s/k8s.py

```
@@ -1,12 +1,12 @@
 import time
-import subprocess
 import asyncio
-import os
 from kubernetes import client
 from kubernetes.client.rest import ApiException
 from workload.pod import create_pod
 from workload.pod import delete_pod
 from workload.pod import new_pod_manifest
+from workload.pod import wait_for_pod_status
```

Ruff

`workload.pod.wait_for_pod_status` imported but unused

Remove unused import: `workload.pod.wait_for_pod_status`

(F401)